tanner/web/server.py

```diff
@@ -90,13 +90,12 @@ def __init__(self):
             'pre_val': pre_val
         }
 
-    @aiohttp_jinja2.template('session.html')
     async def handle_session_info(self, request):
         sess_uuid = request.match_info['sess_uuid']
         session = await self.api.return_session_info(sess_uuid)
-        return {
-            'session': session
-        }
+        if session:
+            return aiohttp_jinja2.render_template('session.html', request, {'session': session})
+        return aiohttp_jinja2.render_template('404.html', request, {})
```

Comment:
for me 404 error is a bit misleading, because `don't have last session`!=`can't find last session page`

Reply:
I'm having a bit difficulty resolving this. @afeena what do you suggest?

Reply:
What are you trying and what's the problem?

Reply:
I'm trying to return 404 on empty session error. Other than that I have no idea how to fix this issue.

Reply:
What about returning a empty list, so that `None` is not shown

Reply:
Currently, a dict `{'session': Session instance}` is passed to `session.html` template. Do you suggest return an empty list to display an empty `session.html` page?

Reply:
Actually I was suggesting to not show None at all

Reply:
@gaochua Look [here](https://github.com/mushorg/tanner/blob/develop/tanner/web/templates/index.html#L49). We could avoid this line if there is no session at all.